managedprofiles.go

```diff
@@ -138,7 +138,7 @@ func (*Service) ResourceOperation(deviceName string, deviceResource string, meth
 
 // DeviceResource retrieves the specific DeviceResource instance from cache according to
 // the Device name and Device Resource name
-func (*Service) DeviceResource(deviceName string, deviceResource string, method string) (contract.DeviceResource, bool) {
+func (*Service) DeviceResource(deviceName string, deviceResource string, _ string) (contract.DeviceResource, bool) {
```

Comment:
Is this 3rd parameter require to match an interface? i.e. can't remove it?

Reply:
It is a public API, so it can't be removed for backward compatibility before entering v2 according to my understanding